### DISCUSSION TITLE
What is the difference of pretrain method in 20-minutes Tour to MMPose?

### DISCUSSION THREAD

**Discussion by seon-creator** [open-mmlab/mmpose#2952] [Q&A] [answered]
Hello, I'm using mmpose for detection keypoint project.
I'm studying to understand more about mmpose usage.
When I read the description about model Backbone part, I checked there are two method of using pretrain model.
![image](https://github.com/open-mmlab/mmpose/assets/118324988/f3a7baef-dde4-4c3a-a8e5-1c4d9075cc32)

So I tried both method on same model with my custom data.
My custom data has 5-keypoint, and I used 2d body model.
This two plots are the train and validation result of loss and AP.
![Train3](https://github.com/open-mmlab/mmpose/assets/118324988/50e354d6-10af-45bb-b5bf-2e546aaedaad)
usign method 1

![Train3_pretrain_to_backbone](https://github.com/open-mmlab/mmpose/assets/118324988/b8a0339c-505e-4850-8fdf-99d55b5bd7ef)
using method 2 

The first plot is the result of applying this method.
```
init_cfg=dict(
    type='Pretrained',
    checkpoint='PATH/TO/YOUR_MODEL_WEIGHTS.pth'),
```

The second plot is the result of applying this method.
```
init_cfg=dict(
    type='Pretrained',
    prefix='backbone.',
    checkpoint='PATH/TO/YOUR_CHECKPOINT.pth'),
```

After then, I compared the accuracy of the keypoint. The result of method 2 was better than 1.
I wonder why the method 2's performance was better than 1.
Is there any idea of this?

**Comment by Tau-J** [comment]
Hi @seon-creator , `prefix='backbone.',` means only load pretrained backbone's weights into the model for training, which is the standard usage for transfer learning.

**Reply by seon-creator** [reply]
Thanks for the quick response!
So, Is that mean that the first method loads the entire model including the backbone?
```
init_cfg=dict(
    type='Pretrained',
    checkpoint='PATH/TO/YOUR_MODEL_WEIGHTS.pth'),
```

**Reply by Tau-J** [reply, marked as answer]
Yes
